packages/eas-cli/src/graphql/types/App.ts

```diff
@@ -1,10 +1,5 @@
-import { Fragment } from '../fragment';
-
-export const AppFragment: Fragment = {
-  name: 'app',
-  definition: /* GraphQL*/ `
-    fragment app on App {
+export const AppFragment = /* GraphQL*/ `
```

Comment:
I probably missed this change, but why did we switch from `graphql-tag` to strings with `/* GraphQL */` comments?

Reply:
1. This is a good question.
2. This PR is not to `main` but to `@quin/generateFragTypes`
3. These changes were made in https://github.com/expo/eas-cli/pull/184 and it looks like you haven't seen it yet.

Reply:
i've since changed this to use `gql`. I originally went with the `\*Graphql*/` comment because we rely on the fragment definition to be a string in many places, whereas `gql` returns a DocumentNode object.